frontend/public/components/utils/details-page.tsx

```diff
@@ -91,8 +91,6 @@ export const ResourceSummary: React.FC<ResourceSummaryProps> = ({
           />
         </DetailsItem>
       )}
-      {type ? <dt>{t('details-page~Type')}</dt> : null}
-      {type ? <dd>{type}</dd> : null}
```

Comment:
Why is this removed? If any resource had the `type` it would not work anymore

Reply:
Type was added for "Secrets" alone. No other resources seemed to use this.
any specific type for a resource can be handled using annotations, as I have done for sinkUri.
"type" for Secret has been added to "SecretKind" which is specific to secret.
please refer to this conversation for more information
https://coreos.slack.com/archives/CHGU4P8UU/p1616684514128300

Reply:
You've removed this from the details view. However I do not see that you've added it back in the right column for secrets

Reply:
this was never used by any component. Using this component would result in a bug being observed since it was rendering an object.
`SecretDetails` already receives a `secret.type`. I just updated it to use `SecretKind` instead of `K8sResourceKind`.

https://github.com/openshift/console/blob/dc102a74f4bbd98607c969b67fa13e1f31dbc6b1/frontend/public/components/secret.jsx#L116